From cce4fc14d1ad9c21a993d94d36fe9cc24bc0db0a Mon Sep 17 00:00:00 2001
From: Shaun McCance <shaunm@gnome.org>
Date: Fri, 02 Jul 2010 15:26:24 +0000
Subject: [yelp-application] Fix broken gsettings keyfile setup

1) The root path must end with a "/".
2) The root path was /org/gnome/yelp, but the schema is /apps/yelp
---
diff --git a/src/yelp-application.c b/src/yelp-application.c
index 3c8436d..9a9c621 100644
--- a/src/yelp-application.c
+++ b/src/yelp-application.c
@@ -224,7 +224,7 @@ application_setup (YelpApplication *app)
         gchar *keyfile = g_build_filename (g_get_user_config_dir (),
                                            "yelp", "yelp.cfg",
                                            NULL);
-        priv->backend = g_keyfile_settings_backend_new (keyfile, "/org/gnome/yelp", NULL);
+        priv->backend = g_keyfile_settings_backend_new (keyfile, "/apps/yelp/", "yelp");
         priv->gsettings = g_settings_new_with_backend ("org.gnome.yelp",
                                                        priv->backend);
         g_free (keyfile);
--
cgit v0.8.3.1
